src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -71,6 +72,16 @@ public EditPersonDescriptorBuilder withAddress(String address) {
         return this;
     }
 
+
```

Comment:
Should the constructor of this class be updated as well?

Reply:
Done